### ISSUE TITLE
Dialog visibility in Dark Mode and Windows High Contrast mode

### ISSUE THREAD

**Issue by jnurthen** [adobe/spectrum-web-components#763]
### Expected Behaviour
The user should be able to easily tell that a dialog is open

### Actual Behaviour
It it is hard to determine if a dialog is open particularly in Dark mode and Windows High Contrast mode

### Reproduce Scenario (including but not limited to)
Dark Mode
Windows High Contrast mode

#### Steps to Reproduce

Run demo https://opensource.adobe.com/spectrum-web-components/components/dialog-wrapper/examples
Enable Dark Mode
Open the Small dialog (toggle dialog button)
Note the color contrast ratio between the background and the dialog is not enough to be able to determine that the dialog is open particularly for low vision users. To meet WCAG there needs to be a 3:1 ratio

Re-run using Windows High Contrast mode and note that in this case the background and the dialog background are the same color. There needs to be a border to distinguish them

#### Platform and Version
Edge Classic (WHCM)
Edge  (WHCM)
Chrome/Safari/Edge for Dark mode

#### Sample Code that illustrates the problem
Dark Mode

![Screen Shot 2020-07-14 at 4 53 16 PM](https://user-images.githubusercontent.com/1724479/87487626-0633bc00-c5f3-11ea-8ca1-603cf2b6d82e.png)

High Contrast Mode
![Spectrum Web Components - Profile 1 - Microsoft​ Edge 7_14_2020 4_57_39 PM](https://user-images.githubusercontent.com/1724479/87487695-35e2c400-c5f3-11ea-9bb9-14d44f98b5eb.png)

**Comment by Westbrook** [comment]
Is this still related to #297 in that this originates upstream? Or have you seen motion of [the issue therein](https://github.com/adobe/spectrum-css/issues/168) that we've yet to adopt?
